files/ko/web/javascript/reference/global_objects/syntaxerror/index.html

```diff
@@ -1,123 +1,110 @@
 ---
 title: SyntaxError
 slug: Web/JavaScript/Reference/Global_Objects/SyntaxError
+tags:
+  - Class
+  - JavaScript
+  - Object
+  - Reference
+  - SyntaxError
 translation_of: Web/JavaScript/Reference/Global_Objects/SyntaxError
+browser-compat: javascript.builtins.SyntaxError
 ---
-<div>{{JSRef}}</div>
-
-<p><code><strong>SyntaxError</strong></code> 객체는 문법적으로 유효하지 않은 코드를 해석하려고 시도할 때 발생하는 오류를 표현합니다.</p>
-
-<h2 id="설명">설명</h2>
-
-<p><code>SyntaxError</code>는 JavaScript 엔진이 코드를 분석할 때 문법을 준수하지 않은 코드를 만나면 발생합니다.</p>
 
-<h2 id="구문">구문</h2>
+<div>{{JSRef}}</div>
 
-<pre class="syntaxbox"><code>new SyntaxError([<var>message</var>[, <var>fileName</var>[, <var>lineNumber</var>]]])</code></pre>
+<p>
+  <code><strong>SyntaxError</strong></code
+  > 객체는 문법적으로 유효하지 않은 코드를 해석하려고 시도할 때 발생하는 오류를
+  의미합니다. Javascript 엔진이 코드를 구문 분석할 때 언어의 구문에 맞지 않는
+  토큰이나 토큰 순서를 만나면 이 오류를 던집니다.
+</p>
 
-<h3 id="매개변수">매개변수</h3>
+<h2 id="생성자">생성자</h2>
 
 <dl>
- <dt><code>message</code><strong> {{optional_inline}}</strong></dt>
- <dd>오류에 대한 설명.</dd>
- <dt><code>fileName</code><strong> {{optional_inline}}</strong> {{non-standard_inline}}</dt>
- <dd>예외가 발생한 코드를 담고 있는 파일의 이름.</dd>
- <dt><code>lineNumber</code><strong> {{optional_inline}}</strong> {{non-standard_inline}}</dt>
- <dd>예외가 발생한 코드의 라인 넘버.</dd>
+  <dt>{{jsxref("Global_Objects/SyntaxError/SyntaxError", "SyntaxError()")}}</dt>
+  <dd>새로운 <code>SyntaxError</code> 객체를 만듭니다.</dd>
 </dl>
 
-<h2 id="속성">속성</h2>
+<h2 id="인스턴스_속성">인스턴스 속성</h2>
 
 <dl>
- <dt>{{jsxref("SyntaxError.prototype")}}</dt>
- <dd><code>SyntaxError</code> 객체에 속성을 추가할 수 있도록 해주고 있습니다.</dd>
+  <dt>
+    {{jsxref("Error.prototype.message", "SyntaxError.prototype.message")}}
+  </dt>
+  <dd>
+    오류 메시지. 비록 ECMA-262에서 {{jsxref("SyntaxError")}}는 반드시 자체
+    <code>message</code> 속성을 제공해야한다고 명시했지만,
+    <a href="/en-US/docs/Mozilla/Projects/SpiderMonkey">SpiderMonkey</a>는
+    {{jsxref("Error.prototype.message")}}를 상속합니다.
+  </dd>
+  <dt>{{jsxref("Error.prototype.name", "SyntaxError.prototype.name")}}</dt>
+  <dd>오류 이름. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>
+    {{jsxref("Error.prototype.fileName", "SyntaxError.prototype.fileName")}}
+  </dt>
+  <dd>이 오류가 발생한 파일 경로. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>
+    {{jsxref("Error.prototype.lineNumber", "SyntaxError.prototype.lineNumber")}}
+  </dt>
+  <dd>
+    이 오류가 발생한 파일의 줄 수. {{jsxref("Error")}}로부터 상속되었습니다.
+  </dd>
+  <dt>
+    {{jsxref("Error.prototype.columnNumber","SyntaxError.prototype.columnNumber")}}
+  </dt>
+  <dd>
+    이 오류가 발생한 줄의 칸 수. {{jsxref("Error")}}로부터 상속되었습니다.
```

Comment:
```suggestion
    이 오류가 발생한 줄(행)에서의 열 위치. {{jsxref("Error")}}로부터 상속되었습니다.
```

`줄의 칸 수`라고 번역할 경우 그 줄의 **총 칸 수**로 해석할 가능성도 있고, `SyntaxError.prototype.lineNumber`와 동일한 의미로 해석할 가능성도 있어보입니다. 위와 같이 바꾸는 게 어떨까요?

Reply:
좋은 의견입니다. 제안하시는 번역이 더 매끄럽네요.